### PR TITLE
feat(sig-services): Add release process guideline

### LINKED ISSUE #223: [SIG Services][Guideline] Release process
Service requires release changelog to include migration guide: removed, deprecated, new features, user facing changes, update propagation timeline.

Release note version mapping to live service. 

Define a release process
- if required have CI/CD pipelines defined to automate the release process
- define test cases that should pass for a new release or change

As stated at https://github.com/open-services-group/community/issues/208 notify user when the feature/bug fix reaches live deployment.

### PR DESCRIPTION
**Pull request by SamoKopecky**
Resolves #223

### CODE REVIEW

File: sig-services/service-requirements/release-process.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+title: Release process
+persona: service provider/owner
+phase: service-design, service-operational
+---
+
+A release process should follow a defined versioning numbering scheme, which means each new version will be identified by a unique number. Every released version should be documented in a changelog.
+
+A changelog contains chronologically ordered list of notable changes for each version of a project. These changes should at least include what was removed, deprecated, added (new features), and fixed. When a new version is released it should map to the live service so that the changes are available to the users. If a release introduces a feature or fixes a bug that was requested/reported by a user via the support process (see the support process guideline), the user should be notified of the change.
+
+Depending on the size and complexity of the service it should also have CI/CD (Continuous Integration/Continuous Deployment) pipelines defined to automate the release process. CI/CD pipelines should also include test cases that must pass before a new version is released to the users.
```
Comment (tumido) [inline sig-services/service-requirements/release-process.md:11]: I think CI/CD is a must have. Especially CD and transparent upgrade path/migration path.

File: sig-services/service-requirements/release-process.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+## Requirements
+
+- Define or reuse a versioning numbering scheme and stick to it
+  - Example of a well-defined scheme: https://semver.org
+- Define a changelog, for example, a markdown document
+  - One possible way of writing a changelog can be found here: https://keepachangelog.com/en/1.0.0
+- If required define CI/CD pipelines including tests
```
Comment (tumido) [inline sig-services/service-requirements/release-process.md:19]: Additionally upgrades should ensure easy/automated migration of users and user settings.

File: sig-services/service-requirements/release-process.md (hunk 1 above)
Comment (SamoKopecky) [reply sig-services/service-requirements/release-process.md:11]: Resolved in the latest commit.

File: sig-services/service-requirements/release-process.md (hunk 2 above)
Comment (SamoKopecky) [reply sig-services/service-requirements/release-process.md:19]: Resolved in the latest commit.

**Comment by SamoKopecky** [comment]
Ready for review again.

**Pull request closed without merging**